### ISSUE TITLE
[Accessibility][ContextMenu]: Disabled items should have aria-disabled attribute set

### ISSUE THREAD

**Issue by prmittal** [OfficeDev/office-ui-fabric-react#1079]
For disabled menu items, aria-disabled attribute should be specified, else screen readers will have trouble reading it clearly. See this: http://www.w3.org/TR/wai-aria/states_and_properties#aria-disabled

Repro:
https://fabricreact.azurewebsites.net/fabric-react/master/#/examples/contextmenu

The html for disabled menu item is as below:
`<li role="menuitem" class="ms-ContextualMenu-item"><button name="Disabled item" disabled="" class="ms-ContextualMenu-link" role="menuitem" aria-label="Disabled item"><div class="ms-ContextualMenu-linkContent"><i class="ms-Icon ms-Icon--undefined ms-ContextualMenu-icon ms-ContextualMenu-iconColor"></i><span class="ms-ContextualMenu-itemText ms-fontWeight-regular">Disabled item</span></div></button></li>`

**Comment by leddie24** [comment]
Closing issue.  Created a PR to revert my previous changes.

From https://www.w3.org/TR/html-aria/

> Use the disabled attribute on any element that is allowed the disabled attribute in HTML5.
> 
> Only use the aria-disabled attribute for elements that are not allowed to have a disabled attribute in HTML5
